packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -219,6 +219,16 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   /* @conditional-compile-remove(file-sharing-acs) */
   const attachmentUploadButtonOnChange = useCallback(
     (files: FileList | null): void => {
+      // look up sendbox by ID for now, we will use `useRef`
```

Comment:
Just wanted to confirm that a user story or task is created for this, os we won't forget about it

Reply:
good call, i will create one, thanks!

Reply:
work item #3764245 created!